### PR TITLE
Added authorization, Login and Signup.

### PR DESCRIPTION
**Pull request by surbhi84**
Added auth i.e login and signup. You can login with default email and password or add a new account with new details.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -466,47 +466,59 @@ Ensure the default browser behavior of the `hidden` attribute.
...
 
 .m-2 {
   margin: 0.5rem;
 }
 
+.m-4 {
+  margin: 1rem;
+}
+
 .my-4 {
```
Comment (aurobindoGupta) [inline src/index.css:481]: better to have a separate file for these

**Review by aurobindoGupta — APPROVED** [review]
nice work 
clean code.
well-defined components.

File: src/components/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                  checked={isChecked}
+                  onChange={() => {
+                    setIsChecked((p) => !p);
+                  }}
+                  className="accent-red-200"
+                />
+                Accept terms & conditions
+              </label>
+              <p>Forgot password?</p>
+            </div>
```
Comment (ishitakeshawani) [inline src/components/Signup.jsx:133]: Here you can put it to the link to do login, for example 
Already have an account?

**Review by ishitakeshawani — COMMENTED** [review]
The eye icon is going outside of the box on the signup page. 
Liked the folder for hooks.
I tried to do a login and sign up but did not work for me.
image
image

**Comment by surbhi84** [comment]
> The eye icon is going outside of the box on the signup page. Liked the folder for hooks. I tried to do a login and sign up but did not work for me. image image

I checked and its breaking in chrome  for smaller resolutions, thanks for letting me know i missed to check my website for every browser.
also i forgot to add the environment variables thatswhy, you couldn't log in.
I have fixed the login issue
https://glistening-stardust-e034ec.netlify.app/
you can check here.

styling might take some time.
Thanks a lot again for pointing out the issues.

**Review by ishitakeshawani — APPROVED** [review]

**Comment by ishitakeshawani** [comment]
I liked toasts and authentication working now.
login and signup UI is also fixed.

**Pull request merged**